src/Microsoft.Azure.WebJobs.Host/Bindings/ConverterManager.cs

```diff
@@ -25,6 +25,7 @@ internal class ConverterManager : IConverterManager
         public ConverterManager()
         {
             this.AddConverter<byte[], string>(DefaultByteArrayToString);
+            this.AddConverter<IEnumerable<JObject>, JArray>((enumerable) => JArray.FromObject(enumerable));
```

Comment:
>AddConverter [](start = 17, length = 12)

This is good - one catch is that converters don't automatically chain. So the TryGet logic below may need to do a special check to chain this case.